NitroxClient/Logger/ClientLogger.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxClient.Logger
+{
+    public class ClientLogger
+    {
+        public const int LOG_CONSOLE = 1;
+        public const int LOG_ERRORMESSAGE = 2;
+        public const int LOG_DEBUG = 4;
+        private static int logLocations = LOG_ERRORMESSAGE;
+
+        // Set with combinaition of constants -- setLogLocation(LOG_CONSOLE | LOG_ERRORMESSAGE)
```

Comment:
I agree that bitmasking is weird. I was conflicted between a boolean for each option and the bitmasking. I just went with bitmasking because it was a bit more condensed, which obviously isn't always better. My thought on this is to allow any combination of outputs, since you might add more options later.
I'll swap this over to methods for setting each individual option.